class AlreadyInDBError(Exception):
    pass

class CorruptDatabaseError(Exception):
    pass
